### PR TITLE
Setup/distribution + HTML viewer

### PR DESCRIPTION
**Pull request by grumpyoldman-io**
**What does this PR do?**

- [x] Added robots.txt with disallow on all.
- [x] Added a distribution builder.
- [x] Added a HTML viewer panel

**Where should the reviewer start?**
`yarn build` should now also output a `distribution` folder in the build folder with a `feed.json` and a `index.html` together with a distrubution zip file.

in storybook (`yarn start`) a HTML panel should be visible

**How should this be tested?**
on the hosting you should be able to go to the following urls:
`https://website.com/distribution`
`https://website.com/distribution/feed.json`

and a zipfile should be downloadable.

**Any background context you want to provide?**
this is untested on netlify.

**Definition of Done (remove if not applicable):**

- [x] I haven't used `--no-verify`
- [x] There is no offensive mock data in this PR.

### CODE REVIEW

**Comment by grumpyoldman-io** [comment]
@Rilliano can you review this please?

File: scripts/build.js [hunk 1]
```diff
@@ -24,25 +29,118 @@ const walk = function(dir) {
...
+    })
+
+    archive.on('error', function(err) {
+      reject(err)
+    })
+
+    archive.pipe(output)
+
+    // Add Components to Archive
+    const allFiles = walk('./src/components')
```
Comment (Rilliano) [inline scripts/build.js]: U can change this into  const allFiles = walk('./src') to retrieve all the files

File: scripts/build.js [hunk 2]
```diff
@@ -24,25 +29,118 @@ const walk = function(dir) {
...
+    const allTwigFiles = allFiles.filter(file => file.substr(-5, 5) === '.twig')
+    allTwigFiles.forEach(file => {
```
Comment (Rilliano) [inline scripts/build.js]: Not only twig files are required in the build, so u can remove the filter and just do:
`allFiles.forEach(file => {
.....
}`

File: scripts/build.js [hunk 3]
```diff
@@ -24,25 +29,118 @@ const walk = function(dir) {
...
+    return `      <div class="stat stat-${stat}">
+        <label style="font-weight:bold;">${stat}:</label>
+        ${stats[stat]}
+      </div>
+`
+  })
+  .join('')}    </div>
+    <a href="${filename}" title="download ${filename}" download>Download</a>
+  </body>
+  </html>`
```
Comment (Rilliano) [inline scripts/build.js]: The HTML code can be indented a little more nicely here

**Review by Rilliano — CHANGES_REQUESTED** [review]
Looks good, just a few remarks that are specific to the needs of mine current project.

It would also be nice if all the components folders consists of a "componentName.exampleData.json" file which can be used to render the "componentName.twig" into a componentName.html file

File: scripts/build.js (hunk 3 above)
Comment (grumpyoldman-io) [reply scripts/build.js]: this is actually the correct way to indent for the output file. see `build/distribution/index.html`

File: scripts/build.js (hunk 2 above)
Comment (grumpyoldman-io) [reply scripts/build.js]: if all files are needed we can replace the `walk` function entirely

File: scripts/build.js (hunk 3 above)
Comment (Rilliano) [reply scripts/build.js]: i've check `build/distribution/index.html` it looks 👌🏼

**Review by Rilliano — APPROVED** [review]
Looks really good!

**Pull request merged**